examples/firmware/Makefile

```diff
@@ -0,0 +1,13 @@
+CC=gcc
+CFLAGS=-Wall
```

Comment:
I'm good with this being a standalone makefile, but consider making CFLAGS more robust.

Maybe `CFLAGS=-Wall -Werror -Wextra` at minimum.